### ISSUE TITLE
The `Location.hash` property is percent-encoded

### ISSUE THREAD

**Issue by yin1999** [mdn/content#37003]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Location/hash

### What specific section or headline is this issue about?

The description at the top of the document

### What information was incorrect, unhelpful, or incomplete?

The `hash` property is also percent-encoded, try to navigate to `https://developer.mozilla.org/en-US/docs/Web/API/Location/hash#%3E`, and see the return value of `location.hash`, it would be `#%3E`, which is encoded.

### What did you expect to see?

Correct the description. The encoding rule is given in the [spec](https://url.spec.whatwg.org/#fragment-percent-encode-set).

### Do you have any supporting links, references, or citations?

See the spec:

- https://url.spec.whatwg.org/#string-percent-decode
- https://url.spec.whatwg.org/#fragment-percent-encode-set

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/location/hash`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Location/hash
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/location/hash/index.md
* Last commit: https://github.com/mdn/content/commit/a2847ff3788f224ffb4cdf05cb0139e07fde7533
* Document last modified: 2024-07-18T19:25:04.000Z

</details>

**Comment by T34-active** [comment]
https://github.com/mdn/content/pull/37022/files

**Comment by wbamberg** [comment]
@yin1999 I was trying to understand how this happened, because sometimes the history gives us context. It looks as if the [original import](https://github.com/mdn/content/blame/4356be9fd2085bdd3cbe4b524478021acbeb9489/files/en-us/web/api/location/hash/index.html) said:

> The fragment is not percent-**decoded**

...then in https://github.com/mdn/content/pull/34855 this got mangled. Does that sound right to you?

**Comment by yin1999** [comment]
> @yin1999 I was trying to understand how this happened, because sometimes the history gives us context. It looks as if the [original import](https://github.com/mdn/content/blame/4356be9fd2085bdd3cbe4b524478021acbeb9489/files/en-us/web/api/location/hash/index.html) said:
> 
> > The fragment is not percent-**decoded**
> 
> ...then in #34855 this got mangled. Does that sound right to you?

Yes, I think using "not precent-decoded" is correct and fully appropriate :)
